plugins/podlet.js

```diff
@@ -18,6 +18,9 @@ export default fp(async function podlet(
     development,
     logger: fastify.log,
   });
+  podlet.defaults({
+    locale,
+  });
```

Comment:
Is this just so that the locale property of the Podium context is correct? I don't see it being used directly either in this PR or in the @podium/element PR right?

Reply:
> 

@digitalsadhu this was added to make sure that the HTML lang attribute was set to the correct value.